### PR TITLE
 GSAGH-338-Add-check-for-installed-versions-of-ComposGH-and-AdSecGH-and-inform-user-of-outdated-versions

### PR DESCRIPTION
**Pull request by tlmnrnhrdt**

### CODE REVIEW

**Review by kpne — APPROVED** [review]
Please review my commits before merging

File: GsaGH/Helpers/Versions.cs [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+
+        case (false, true):
+          text += "ComposGH Plugin";
+          process += ComposGuid;
+          header += "Compos";
+          icon = Properties.Resources.ComposGHUpdate;
+          break;
+
+        default:
+          text = "Updates are avaiable for AdSecGH and ComposGH";
```
Comment (tlmnrnhrdt) [inline GsaGH/Helpers/Versions.cs:59]: typo

File: GsaGH/Helpers/Versions.cs [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+
+    internal static UpdatePluginsBox CreatePluginBox(bool isAdSecOutdated, bool isComposOutdated) { 
+      string process = @"rhino://package/search?name=guid:";
+      string text = "An update is available for ";
+      string header = "Update ";
+      Bitmap icon = Properties.Resources.OasysGHUpdate;
+
+      switch (isAdSecOutdated, isComposOutdated) {
+        case (true, false):
+          text += "AdSecGH Plugin";
```
Comment (tlmnrnhrdt) [inline GsaGH/Helpers/Versions.cs:45]: just `AdSec` instead of `AdSecGH Plugin` ?

File: GsaGH/Helpers/Versions.cs [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+          process += AdSecGuid;
+          header += "AdSec";
+          icon = Properties.Resources.AdSecGHUpdate;
+          break;
+
+        case (false, true):
+          text += "ComposGH Plugin";
```
Comment (tlmnrnhrdt) [inline GsaGH/Helpers/Versions.cs:52]: same here

File: GsaGH/Helpers/Versions.cs (hunk 2 above)
Comment (tlmnrnhrdt) [reply GsaGH/Helpers/Versions.cs:45]: or just `AdSec` without plugin ;)

**Comment by kpne** [comment]
Making this a draft until we decide if it should move to OasysGH

**Comment by kpne** [comment]
Work done in OasysGH: https://github.com/arup-group/OasysGH/pull/97

**Pull request closed without merging**